playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -51,6 +51,9 @@
       '{"spec":{"template":{"spec":{"containers":[{"name":"router","image":"{{ router_image }}","livenessProbe":{"tcpSocket":null,"httpGet":{"path": "/healthz", "port": 1936, "host": "localhost", "scheme": "HTTP"},"initialDelaySeconds":10,"timeoutSeconds":1}}]}}}}'
       --api-version=v1
     with_items: "{{ haproxy_routers }}"
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
Correct. The note says we're not but maybe in the future we should look at this task more. There's a longer discussion around it and similar audit comments here: https://github.com/openshift/openshift-ansible/pull/3334#discussion_r100583870